ekan0ra/queue.py

```diff
@@ -27,13 +27,11 @@ def enqueue(self, nick):
 
     def dequeue(self, nick):
         """
-        Remove first item matching `nick` in queue.
+        Remove every occurrence of user from a queue.
```

Comment:
@rahuldecoded:
Please change:
> Remove every occurrence of **user** from **a** queue.

to:

> Remove every occurrence of **nick** from queue.

And please keep it to one line according to this _[PEP](https://www.python.org/dev/peps/pep-0257/#id16)_.